bundles/admin/admin-layereditor/components/Tabs.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AntTabs from 'antd/lib/tabs';
+import 'antd/lib/tabs/style/css';
+
+const AntTabPane = AntTabs.TabPane;
+
+export const Tabs = (props) => {
+    const {tabPanes, ...other} = props;
```

Comment:
We should be able to first wrap `AntTabs.TabPane` and then use it via `props.children`. So we can have structure: 
```
<Tabs>
  <TabPane />
  <TabPane />
</Tabs>
```

Reply:
Yep, I created TabPane component and modified Tabs to use it.